src/core_plugins/kibana/public/discover/controllers/discover.js

```diff
@@ -292,6 +293,8 @@ function discoverController(
     $state.save();
   });
 
+  $scope.$watch('state.showViz', $state.save.bind($state));
```

Comment:
Nit: Can we use a different variable name, like maybe `isVizVisible` or something? It is a little confusing here that `state.showViz` is a boolean while `$scope.showViz` is a function.

Reply:
How about `isVizVis`? 😆

Reply:
`isVizable`!